node_modules/col-canvas/lib/poller.js

```diff
@@ -680,7 +680,11 @@ var handleAssignmentSubmissions = function(ctx, users, assignment, callback) {
         return callback(err);
       }
 
-      log.info({'course': ctx.course.id, 'assignment': assignment.id}, 'Got ' + _.size(submissions) + ' submissions from Canvas');
+      var activeSubmissions = _.reject(submissions, {'workflow_state': 'unsubmitted'});
```

Comment:
@johncrossman I also don't know what 'pending' means. Since 'unsubmitted' appears to be what's causing the current fire, I decided to douse selectively.